sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -128,6 +128,11 @@ public String getType() {
     }
 
     @Override
+    public void setUser(String user) {
```

Comment:
instead of adding a setter, have the calling code create a new builder from the current copy, edit the user in the builder, and then create a new copy with the user set.